### ISSUE TITLE
Conflicting statements in “Mobile accessibility checklist” article

### ISSUE THREAD

**Issue by AurelBouchard** [mdn/content#5184]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Mobile_accessibility_checklist#general_guidelines

(French version too): https://developer.mozilla.org/fr/docs/Web/Accessibility/Mobile_accessibility_checklist#general_guidelines

#### What information was incorrect, unhelpful, or incomplete?
in paragraph:
- The down-event should not be used to trigger any action
- The action is triggered on the up event and an option to abort the action before its completion is available or an option to undo the action after its completion
- The up-event will undo any action that was triggered on a down event
- It is essential to trigger the action on the down event. For example, playing a game or a piano application

(also in French version):
- L'événement de descente NE DOIT PAS être utilisé pour déclencher une action.
- L'action est déclenchée par l'événement « up » (...) achèvement.
- L'événement de montée annulera toute action déclenchée par un événement de descente.
- Il est essentiel de déclencher l'action sur l'événement de descente. Par exemple, pour jouer à un jeu ou à une application de piano.

=> the last line is the opposite meaning of the first one, no translation error, the english page have the same issue

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `fr/web/accessibility/mobile_accessibility_checklist`
* MDN URL: https://developer.mozilla.org/fr/docs/Web/Accessibility/Mobile_accessibility_checklist
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/fr/web/accessibility/mobile_accessibility_checklist/index.html
* Last commit: https://github.com/mdn/translated-content/commit/4d3b1c2c79113786e001e3bec551205cd34f818a
* Document last modified: 2021-05-21T00:42:40.000Z

</details>

**Comment by himanshugarg** [comment]
Yes the english page has these seemingly conflicting first and fourth bullets :-

- The down-event should not be used to trigger any action
- The action is triggered on the up event and an option to abort the action before its completion is available or an option to undo the action after its completion
- The up-event will undo any action that was triggered on a down event
- It is essential to trigger the action on the down event. For example, playing a game or a piano application

**Comment by dontcallmedom** [comment]
I agree the MDN text is confusing (in both languages); the [intent from the original spec](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html#down-event) is that relying on the down-event be reserved to cases where there is no alternative, which MDN should convey in a simpler language than what the spec has to use.

**Comment by dawei-wang** [comment]
I propose changing the first bullet to read: "The down-event should not be used to trigger any action unless there is no alternative. For example when using an on-screen keyboard or playing a game." and removing the fourth bullet.

**Comment by hamishwillee** [comment]
The list is preceded with the text "For touch events, at least one of the following must be true (WCAG 2.1: Pointer Cancellation)". It is therefore correct as written,  but as @dontcallmedom states above, not particularly simple or easy to understand.

 @dawei-wang This means that you can't make the changes suggested unless you also modify the statement (i.e. if you delete the last point you make it incorrect).
 
 The best way to progress this would be for someone to rewrite the section as a PR so we could look at the resulting text in context.
